CrowdStrike/manifest.json

```diff
@@ -29,5 +29,5 @@
     "name": "CrowdStrike",
     "uuid": "4ffe6bd9-6693-414d-ade0-5ec9fb1b8b2c",
     "slug": "crowdstrike",
-    "version": "1.9.1"
+    "version": "1.9.2"
```

Comment:
For the others module you updated the minor version

Reply:
Indeed, I have harmonized :)